packages/block-library/src/site-logo/block.json

```diff
@@ -33,6 +33,20 @@
 		"html": false,
 		"align": true,
 		"alignWide": false,
+		"__experimentalBorder": {
+			"color": true,
+			"radius": true,
+			"width": true,
+			"style": true,
+			"__experimentalSelector": "img, .components-placeholder__illustration, .components-placeholder::before, .wp-block-image__crop-area",
```

Comment:
This should use the [block selectors API](https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-selectors.md) rather than the outdated `experimentalSelector` property, as per the [image block](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/image/block.json#L122-L127).

Reply:
If I try to move both the duotone and the border to selectors, the duotone changes the color of the resize handles. I am not sure how that is prevented in the image block.

Reply:
I'm getting confused, the more I test this, the more it looks like the selector for the border is not even needed.

Reply:
Does the change below not work? Personally, I have submitted #58035 regarding the deprecated duotone selector, so I think it can be removed from the scope of this PR.

```diff
diff --git a/packages/block-library/src/site-logo/block.json b/packages/block-library/src/site-logo/block.json
index 977a042a72..44c08e60da 100644
--- a/packages/block-library/src/site-logo/block.json
+++ b/packages/block-library/src/site-logo/block.json
@@ -38,7 +38,6 @@
                        "radius": true,
                        "width": true,
                        "style": true,
-                       "__experimentalSelector": "img, .components-placeholder__illustration, .components-placeholder::before, .wp-block-image__crop-area",
                        "__experimentalSkipSerialization": true,
                        "__experimentalDefaultControls": {
                                "color": true,
@@ -48,7 +47,6 @@
                        }
                },
                "color": {
-                       "__experimentalDuotone": "img, .components-placeholder__illustration, .components-placeholder::before",
                        "text": false,
                        "background": false
                },
@@ -59,6 +57,15 @@
                                "margin": false,
                                "padding": false
                        }
+               },
+               "filter": {
+                       "duotone": true
+               }
+       },
+       "selectors": {
+               "border": ".wp-block-site-logo img, .components-placeholder__illustration, .wp-block-site-logo .components-placeholder::before, .wp-block-site-logo .wp-b
lock-image__crop-area",
+               "filter": {
+                       "duotone": ".wp-block-site-logo img, .wp-block-site-logo .components-placeholder__illustration, .wp-block-site-logo .components-placeholder::befo
re"
                }
        },
        "styles": [
```

Reply:
I had set 	color :` "__experimentalDuotone": true`, that's what was affecting the resize handlers.

Reply:
> I don't see anything breaking.

If you simply remove `__experimentalBorder.__experimentalSelector`, the border functionality at the block level may still work. However, when applying a border to the Site Logo block via the Global Styles, the border will not be applied to the intended element. My understanding is that this property solves this, and I think it's the intended behavior.

Reply:
OK I think I'm having one of those days where I shouldn't be close to a computer 😁